includes/sanitizers/class-amp-nav-menu-toggle-sanitizer.php

```diff
@@ -99,6 +99,11 @@ public function sanitize() {
 		$script_el->appendChild( $this->dom->createTextNode( wp_json_encode( $expanded ) ) );
 		$state_el->appendChild( $script_el );
 		$nav_el->parentNode->insertBefore( $state_el, $nav_el );
+		if ( 'body' === $nav_el->nodeName ) {
```

Comment:
FWIW I've used the `html` element in the past for toggle classes. Can we test against this as well?

Speaking of testing... Are there unit tests for that?

Reply:
I was worried about that as well, but we tested it together and it didn't seem to cause any problem. 

In particular, I was worried about classes that AMP adds dynamically, including `amp-mode-mouse` or any of the classes that `amp-geo` adds. However, it appears that AMP is smart enough to persist such class names even when mutating the class via `amp-bind`. See:

https://github.com/ampproject/amphtml/blob/c2cf35069cd740822c99538830d29aedceaced12/extensions/amp-bind/0.1/bind-impl.js#L1155-L1173

A safer solution I think would actually be to make use of AMP's [`toggleClass()` action](https://amp.dev/documentation/guides-and-tutorials/learn/amp-actions-and-events#*-(all-elements)), although this would depend on (1) the container having an ID and (2) that we support `toggleClass` in the tree shaker (https://github.com/ampproject/amp-wp/issues/1820). This would ensure that multiple components could toggle classes on the `body` without stomping on each other.

Reply:
So it works even when using `html` and not just `body`?

Reply:
Oh, that's a good point.

Reply:
@felixarntz if `$nav_el` is the `$this->dom->documentElement` then the `amp-state` needs to be added to `body`, not the `$nav_el`.